tests/integration/components/search-result/component-test.js

```diff
@@ -1,25 +1,92 @@
+import Ember from 'ember';
 import { moduleForComponent, test } from 'ember-qunit';
-import hbs from 'htmlbars-inline-precompile';
 
 moduleForComponent('search-result', 'Integration | Component | search result', {
-  integration: true
+    integration: true,
+    beforeEach: function() {
+        let noop = () => {};
+        this.set('noop', noop);
+    }
 });
 
+let result = Ember.Object.extend({
```

Comment:
I might be misunderstanding what you mean, but I tried to make it a factory under test_support and was getting an error that it wasn't a model.